front-end/src/components/toolbar/SortingBar.js

```diff
@@ -7,17 +7,15 @@ export function SortingBar(props) {
   return (
     <ButtonToolbar>
       <ToggleButtonGroup type="radio" name="options" defaultValue={1}>
-        <ToggleButton
-          variant="light"
-          onClick={props.onClick}
-          prechecked
-          value={1}
-        >
+        <ToggleButton variant="light" onClick={props.onClick} prechecked value={1}>
           Sort by number of mentees
         </ToggleButton>
         <ToggleButton variant="light" onClick={props.onClick1} value={2}>
           Sort by mentor
         </ToggleButton>
+        <ToggleButton variant="light" onClick={props.onClick2} value={3}>
+          Sort by last met
```

Comment:
"Sort by last meeting date" i think might be better

Reply:
Or actually, we could bring in line more with the mockup, where we have a label:
"Sort by:"
Then each button..

Reply:
done